db/migrate/20170802201510_create_scholarships.rb

```diff
@@ -0,0 +1,14 @@
+class CreateScholarships < ActiveRecord::Migration[5.0]
+  def change
+    create_table :scholarships do |t|
+      t.string :name
+      t.text :description
+      t.string :location
+      t.text :terms
+      t.datetime :start_date
```

Comment:
I think date time is better since we can say something like 'Applications close at 5pm 8/2'. Maybe a better name is in order? open_time and close_time?